### DISCUSSION TITLE
Ubuntu network not showing in UTM

### DISCUSSION THREAD

**Discussion by TheGuyWhoLikesUbuntu** [utmapp/UTM#3929] [UTM Help (macOS)]
So I created a fresh VM with Ubuntu on my M1 Macbook pro and I realized that I had no network. I don’t know why I tried emulated Vlan , bridged and shared It doesn’t work. I tried to change the network card but didn’t work. Please help I appreciate.

**Configuration**
* UTM Version: I think the latest
* OS Version: MacOS Monetary/ VM: Ubuntu 20.04.4 LTS
* Apple Silicon

**Comment by KendrickLamar2003** [comment]
Have you tried checking your network settings on your MacBook?

**Comment by TheGuyWhoLikesUbuntu** [comment]
> Have you tried checking your network settings on your MacBook?

Yes it works perfectly on my Mac but not on the VM

**Comment by welcomeyou2019** [comment]
I have the same problem.

**Comment by osy** [comment]
Please refer to troubleshooting https://mac.getutm.app/gallery/ubuntu-20-04
